### ISSUE TITLE
page.attach_file options aren't listed in documentation

### ISSUE THREAD

**Issue by alextaylor000** [jnicklas/capybara#1659]
The documentation for `Capybara::Node::Actions#attach_file` doesn't specify the options that can be passed to it - for example, passing `:match => :first` to get the first match in the case of ambiguous selectors.

I'm happy to submit a PR updating the relevant docs, I just wanted to confirm that the valid options are whatever is valid for `Class: Capybara::Query`, i.e.:

```
VALID_KEYS = [:text, :visible, :between, :count, :maximum, :minimum, :exact, :match, :wait]
VALID_MATCH = [:first, :smart, :prefer_exact, :one]
```

Thanks :)

**Comment by twalpole** [comment]
The allowed options would be the ones allowed by #find plus any extras added by the :file_field selector however some don't really make any sense - :text for instance since a file field doesn't have text content, the count ones (count, maximum, minimum, between) don't really make any sense in this context - so I guess the full list of options that may make some sense is :match, :exact, :wait, :disabled, :multiple

**Comment by alextaylor000** [comment]
Great, thanks @twalpole! PR is cookin'.

**Comment by twalpole** [comment]
Closed through PR #1660
